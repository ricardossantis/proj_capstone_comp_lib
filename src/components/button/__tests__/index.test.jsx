import React from 'react'
import Button from '../index'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const testButton = renderer.create(
    (<Button onClick={} text={false} />).toJSON()
  )

  expect(testButton).toMatchSnapshot()
})
