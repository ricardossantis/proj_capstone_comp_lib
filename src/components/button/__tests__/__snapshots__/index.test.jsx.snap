// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="sc-bdnylx gbrgUp"
  onClick={[Function]}
>
  Click to submit
</button>
`;

exports[`renders well 1`] = `
<button
  className="sc-gtssRu ftMiYT"
  onClick={[Function]}
>
  Click to cancel
</button>
`;
